internal/pkg/template/artifactpath/artifactpath.go

```diff
@@ -38,6 +39,12 @@ func AddonAsset(workloadName, hash string) string {
 	return path.Join(s3ArtifactDirName, s3ArtifactAddonsDirName, workloadName, s3ArtifactAddonAssetDirName, hash)
 }
 
+// EnvironmentAddonAsset returns the path to store an addon asset file for an environment addon.
+// Example: manual/addons/environments/assets/668e2b73ac.
```

Comment:
I don't know why we put `assets` in there 😂 for workloads - seems redundant 🤔

Reply:
I think it's storing the packaged content inside of an addon, not the addon itself. Hence "assets" to addons.